kitchen/wazuh-puppet/Dockerfile

```diff
@@ -74,6 +71,19 @@ RUN gem install rake \
       --no-document \
       --quiet
 
+# Clone wazuh-qa
+RUN cd $HOME && \
+    git clone https://github.com/wazuh/wazuh-qa.git && \
+    cd $HOME/wazuh-qa && \
+    git fetch --all && \
+    git checkout devel && \
+    git pull origin devel
```

Comment:
No need to use `pull` after `fetch --all`.

Reply:
Hi @manuasir, 

I removed `git fetch --all` so the task to run, now is
```Dockerfile
RUN cd $HOME && \
    git clone https://github.com/wazuh/wazuh-qa.git && \
    cd $HOME/wazuh-qa && \
    git checkout devel && \
    git pull origin devel
``` 

I think `pull` is needed so it combines the `fetch` of remote changes and `merge` them in local, in one command.

Done in https://github.com/wazuh/wazuh-qa/pull/631/commits/20aa885b206c78ca7c4f9edcf6c517206d03cab6.

Kr,

Rshad